src/python/grpcio/commands.py

```diff
@@ -281,6 +282,18 @@ def get_ext_filename(self, ext_name):
         return filename
 
     def build_extensions(self):
+
+        use_short_temp = os.environ.get("GRPC_USE_SHORT_BUILD_TEMP", 0)
```

Comment:
Please add a note explaining why we added this workaround. Create a ticket to rid of this tech debt, link here.

Reply:
Thank you!